stats/cloud/collector.go

```diff
@@ -527,16 +556,48 @@ func (c *Collector) pushMetrics() {
 	c.bufferSamples = nil
 	c.bufferMutex.Unlock()
 
+	count := len(buffer)
 	logrus.WithFields(logrus.Fields{
-		"samples": len(buffer),
+		"samples": count,
 	}).Debug("Pushing metrics to cloud")
+	start := time.Now()
+
+	numberOfPackages := ceilDiv(len(buffer), int(c.config.MaxMetricSamplesPerPackage.Int64))
```

Comment:
I guess this is my not so good English, but for me, `package` and `packet` are synonyms ;). I would've agreed to change it to `packet` ... a year and a half ago when it was added, but now I don't think it is a good idea.

Also `message` to me sounds, not ... right in the context of HTTP requests ... (I guess `request` would've been fine as well), but in GRPC, if we ever change to it, it will probably be better.

The problem is that, for me, the point is moot, as I don't think changing it adds enough value for the breaking change and being wrong in the naming is bad, but being wrong and inconsistent is worse - so I am also against it having different names in the code and in the configuration, at least at this time